test/tests/functional/pbs_acl_host_moms.py

```diff
@@ -94,13 +97,68 @@ def test_acl_host_moms_enable(self):
         ret = self.du.run_cmd(self.remote_host, cmd=self.pbsnodes_cmd)
         self.assertEqual(ret['rc'], 0)
 
+        ret = self.du.run_cmd(self.remote_host, cmd=self.qstat_cmd)
+        self.assertEqual(ret['rc'], 0)
+
     def test_acl_host_moms_disable(self):
         """
         Set acl_host_moms_enable = False and check whether or not the remote
-        host is forbidden to run pbsnodes.
+        host is forbidden to run pbsnodes and qstat.
         """
         self.server.manager(MGR_CMD_SET, SERVER, {
                             'acl_host_moms_enable': False}, expect=True)
 
         ret = self.du.run_cmd(self.remote_host, cmd=self.pbsnodes_cmd)
-        self.assertEqual(ret['rc'], 1)
+        self.assertNotEqual(ret['rc'], 0)
+
+        ret = self.du.run_cmd(self.remote_host, cmd=self.qstat_cmd)
+        self.assertNotEqual(ret['rc'], 0)
+
+    def test_acl_host_moms_hooks_and_jobs(self):
+        """
+        Use hooks to test whether remote host is able to run pbs.server()
+        and check whether the job that is submitted goes to the 'R' state.
+        """
+        hook_name = "hook_acl_host_moms"
+        hook_body = """
+import pbs
+e = pbs.event()
+svr = pbs.server()
+e.accept()
+"""
+        a = {'event': 'queuejob', 'enabled': 'True'}
+        self.server.create_import_hook(
+            hook_name, a, hook_body, overwrite=True)
+
+        self.server.manager(MGR_CMD_SET, SERVER, {
```

Comment:
Just a suggestion: I think this test will better demonstrate/test the acl_host_moms_enable attribute if you first submitted a job with acl_host_moms_enable=False, show how the queue-job hook fails, then set it to True and show that it passes. What do you think?